syntax = "proto2";

package recursive_dependency_embedding;

import "sequence_node.proto";

message SimilarityTreeTuple {
    required float similarity = 1;
    required SequenceNode first = 2;
    required SequenceNode second = 3;
}